vackup

```diff
@@ -34,9 +34,12 @@ save/load copies files between a container image and a container volume.
 
 Usage:
 
-vackup export VOLUME FILE
+vackup export [OPTIONS] VOLUME FILE
   Creates a gzip'ed tarball in current directory from a container volume
 
+  Options:
+    -o, --owner=UID[:GID]        Change the ownership of the the exported archive file
```

Comment:
1. We got rid of `-o` right?
2. This only works on Linux. On macOS Docker Desktop, permissions of export will always be of the mac user running Docker. maybe change to `Change the ownership of the exported archive on Linux`
3. Can you also add an example to the README on using `--owner`?

Reply:
1. Nice catch, missed that 🤦
2. Is there really no way to make it work on Mac/BSD? I don't have anything Apple, so I don't know and can't test that.
3. Sure.